src/main/java/com/alipay/oceanbase/hbase/filter/HBaseFilterUtils.java

```diff
@@ -133,9 +133,14 @@ private static String toParseableString(WhileMatchFilter filter) {
     private static String toParseableString(FilterList filterList) {
         StringBuilder sb = new StringBuilder();
         List<Filter> filters = filterList.getFilters();
-        if (!filters.isEmpty()) {
-            sb.append("(").append(toParseableString(filters.get(0)));
-            for (int i = 1; i < filters.size(); i++) {
+        boolean isFirst = true;
```

Comment:
would you rename isFirst to isEmpty, which is more understandable?

Reply:
Fixed